### DISCUSSION TITLE
Should the debug adapter for node's instructions be changed since the repo is now archived?

### DISCUSSION THREAD

**Discussion by joshpetit** [mfussenegger/nvim-dap#480] [General]
Looking at [vscode node debug 2](https://github.com/microsoft/vscode-node-debug2), it's archived in favor of [this](https://github.com/microsoft/vscode-js-debug).

Should we change the instructions in the wiki to configure it to reflect this?

**Comment by mfussenegger** [comment]
As far as I am aware vscode-js-debug still doesn't work without custom extensions

**Reply by joshpetit** [reply]
Oh I'm sorry, is this something that's been discussed elsewhere in an issue?
